iroh-gateway/src/headers.rs

```diff
@@ -61,7 +61,6 @@ pub fn add_cache_control_headers(headers: &mut HeaderMap, metadata: Metadata) {
             HeaderValue::from_str(&lmdt.to_string()).unwrap(),
         );
     } else {
-        headers.insert(LAST_MODIFIED, HeaderValue::from_str("0").unwrap());
```

Comment:
@Arqu @faassen what's the deal here?

Reply:
I think I might've squashed one too many a commit or something.

Reply:
Ah, it *doesn't* show up in the diff, only in the commits. Must be me bungling fetching upstream or something.